packages/destination-actions/src/destinations/snap-conversions-api/snap-capi-properties.ts

```diff
@@ -351,7 +351,7 @@ export const formatPayload = (payload: Payload): Object => {
     event_conversion_type: payload?.event_conversion_type,
     event_tag: payload?.event_tag,
     timestamp: Date.parse(payload?.timestamp),
-    hashed_email: hash(payload?.email),
+    hashed_email: new RegExp(/[0-9abcdef]{64}/gi).test(String(payload?.email)) ? payload?.email : hash(payload?.email),
```

Comment:
Could we move the regex into a method to keep the code readable?
```suggestion
    hashed_email: isHashedEmail(payload?.email) ? payload?.email : hash(payload?.email),
```